test/regression/cases/file_to_blackhole/experiment.yaml

```diff
@@ -5,13 +5,16 @@ environment:
   DD_TELEMETRY_ENABLED: true
 
 profiling_environment:
-  DD_INTERNAL_PROFILING_ENABLED: true
-  DD_INTERNAL_PROFILING_UNIX_SOCKET: /var/run/datadog/apm.socket
+  DD_INTERNAL_PROFILING_BLOCK_PROFILE_RATE: 10000
+  DD_INTERNAL_PROFILING_CPUDURATION: 1m
```

Comment:
These should be `_CPU_DURATION`

Reply:
[fade466](https://github.com/DataDog/datadog-agent/pull/20081/commits/fade466e911310e6d253784f2e3a79fa6e2d0fe4)